entries/g/gyazo.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Gyazo": {
+    "domain": "gyazo.com",
+    "tfa": [
+      "email"
+    ],
+    "documentation": "https://help.gyazo.com/--5ed8969fc1eca80023e84383",
+    "categories": [
+      "creativity"
```

From what I understand, Gyazo is a file sharing service, akin to Dropbox. If that's the case, _Backup and Sync_ would be a better fit.
```suggestion
      "backup"
```

---

@Carlgo11 I see. Please forgive me for not being familiar with website classification. I thought this website was a service for sharing creative inspiration. However, I took a closer look and you are right. We are currently researching websites about 2FA, so we have sorted out a lot of websites. I am not very familiar with Github submissions, so thank you for correcting me in time! I have committed it again, hope your review!